generators/app/package-json.js

```diff
@@ -124,18 +124,16 @@ const packageJSON = (current, context) => {
       'ie 11'
     ],
     'scripts': _.assign({}, current.scripts, {
-      'prebuild': 'npm run clean',
-      'build': 'npm-run-all -p build:*',
+      'build-test': "npm-run-all -s clean -p 'build:!(js)' 'build:js -- --environment TEST_BUNDLE_ONLY'",
+      'build-prod': "npm-run-all -s clean -p 'build:!(js)' 'build:js -- --environment NO_TEST_BUNDLE'",
```

Comment:
`build:!(js)` is run all build commands except for js, then we run it separately with an environment variable.

Reply:
I didn't know npm-run-all supported segation like that, nice.